packages/salesforcedx-vscode-core/package.json

```diff
@@ -75,12 +75,28 @@
   ],
   "main": "./out/src",
   "contributes": {
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "metadata",
+          "title": "Org Browser",
```

Comment:
![Screen Shot 2019-05-01 at 4 16 35 PM](https://user-images.githubusercontent.com/10779873/57049348-8f4c1680-6c2c-11e9-97f4-46f8f7c3f987.png)
For reference this is where the title shows @ruthemmanuelle @ntotten